.github/workflows/test.yml

```diff
@@ -0,0 +1,135 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    name: Elixir ${{matrix.elixir}} / OTP ${{matrix.otp}}
+    runs-on: ubuntu-18.04
+
+    services:
+      postgres:
+        image: postgres
+        ports:
+        - 5432:5432
+        env:
+          POSTGRES_DB: dataloader_test
+          POSTGRES_PASSWORD: postgres
+
+    strategy:
+      matrix:
+        elixir:
+          - '1.7'
+          - '1.8'
+          - '1.9'
```

Comment:
Absinthe tends to only support the last 2 Elixir versions, so we can remove 1.7-1.9. This does need to be updated in our `mix.exs` probably.

Reply:
Done.